include/dsVideoPort.h

```diff
@@ -941,7 +933,10 @@ dsError_t dsGetHdmiPreference(intptr_t handle, dsHdcpProtocolVersion_t *hdcpCurr
 /**
  * @brief Gets the IgnoreEDID status variable set in the device.
  *
- * This function is used to retrieve the status variable in order to determine whether to ignore the EDID data or not.
+ * For sink devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
+ * For source devices, this function is used to retrieve the status variable in order to determine 
+ * whether to ignore the EDID data or not. Used by dsMgr to decide whether it should handle the hdmi resolution settings 
```

Comment:
we should not refer any caller references like dsMgr. 
Rephrase like below,
"This will help the caller whether it should handle the hdmi resolution settings or not after hdcp authentication. If false, caller handles the settings else by the SoC."

Reply:
Updated